handbook/engineering/distribution/goals_completed.md

```diff
@@ -0,0 +1,118 @@
+# Distribution team completed goals
+
+This page records goals completed by the Distribution team historically. See also our [current goals](goals.md).
+
+- **2020**
+  - [FY-20-Q4](#FY-20-Q1)
+  - [FY-20-Q3](#FY-20-Q1)
+  - [FY-20-Q2](#FY-20-Q1)
+  - [FY-20-Q1](#FY-20-Q1)
+- **2019**
+  - [FY-19-Q4](#FY-19-Q4)
+
+## FY-20-Q4
+
+### Automatic e2e testing
+
+[RFC 137](https://docs.google.com/document/d/14f7lwfToeT6t_vxnGsCuXqf3QcB5GRZ2Zoy6kYqBAIQ/edit#heading=h.trqab8y0kufp) saw us remove our e2e tests from CI entirely because it was unreliable. Now e2e tests are ran as part of our monthly release process completely manually, and are heavily broken/outdated each time we attempt to do it. Fixing them often takes ~1.5d of work from a developer on the team. Per the RFC, we want to run these e2e tests on CI in an automated and reliable fashion.
```

Comment:
```suggestion
[RFC 137](https://docs.google.com/document/d/14f7lwfToeT6t_vxnGsCuXqf3QcB5GRZ2Zoy6kYqBAIQ/edit#heading=h.trqab8y0kufp) saw us remove our e2e tests from CI entirely because it was unreliable. Now e2e tests are run as part of our monthly release process completely manually, and are heavily broken/outdated each time we attempt to do it. Fixing them often takes ~1.5d of work from a developer on the team. Per the RFC, we want to run these e2e tests on CI in an automated and reliable fashion.
```